clients/src/main/java/org/apache/kafka/common/protocol/ProtoUtils.java

```diff
@@ -40,6 +40,12 @@ public static short latestVersion(int apiKey) {
         return Protocol.CURR_VERSION[apiKey];
     }
 
+    public static short oldestVersion(int apiKey) {
```

Comment:
On second thought, I'm OK to leave this as it is and we can leave this improvement as future work.